Sources/Contribute/FileURLDownloader.swift

```diff
@@ -1,38 +1,126 @@
+// swiftlint
 import Foundation
 
 #if canImport(FoundationNetworking)
   import FoundationNetworking
 #endif
+
+// TODO: Update this description
+/// A struct that downloads files from URLs using the `URLSession` class.
```

Comment:
I have some ideas on better architecture for this. Don't do anything now but note it somewhere